sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -223,6 +223,44 @@ export interface PeekMessagesOptions extends OperationOptionsBase {
     fromSequenceNumber?: Long;
 }
 
+// @public
+export interface ProcessErrorContext {
+    entityPath: string;
+    errorSource: "complete"
+    /**
+     * Covers errors that occur when the user (or autoComplete) abandons a message.
+     */
+     | "abandon"
+    /**
+     * Errors thrown from the user's `processMessage` callback passed to `subscribe`
+     */
+     | "userCallback"
```

Comment:
why not call it "processMessage"?

Reply:
@JoshLove-msft  - any objection to this? I pulled this value from your code - we don't have to match but perhaps you like this name better?

Reply:
@HarshaNalluru  - as another point to this, I do have one other spot where I use this (it's for processInitialize) but that's an undocumented internal option at the moment.

Reply:
I like `userCallback` more because it is much more clear than `processMessage`.
Should this be `messageCallback` instead because the error handler is also a "user callback", but we dont report errors from there

Reply:
No strong opinion on this.

Reply:
userCallback seems pretty clear to me but I don't mind following the name of the method at least (processMessageCallback).

My only thought about 'messageCallback' was it might feel like it applies to messages on the message itself but that wouldn't be true.

Reply:
Thumbs up if you're okay with 'processMessageCallback'

CC: @ramya-rao-a , @HarshaNalluru  since you both suggested slightly similar variations :)

Reply:
Renamed to processMessageCallback